Package.swift

```diff
@@ -3,6 +3,6 @@ import PackageDescription
 let package = Package(
     name: "SwiftBitset",
     targets: [
```

Comment:
I think we need to add "swiftLanguageVersions: [4]".

Reply:
https://github.com/apple/swift-evolution/blob/master/proposals/0151-package-manager-swift-language-compatibility-version.md